lib/unique_identifier.rb

```diff
@@ -5,26 +5,32 @@ module UniqueIdentifier
 
   module ClassMethods
 
+    # FIX ME: Are these needed?
     mattr_accessor :field, :block, :klass
 
     def unique_id(field, block)
-      const_set('BLOCK', block)
-      const_set('FIELD', field)
+      const_set('UNIQUE_IDENTIFIER_FIELD', field)
```

Comment:
I like the more explicit constants. Less likely to be overwritten or override other constants.